docker/create_dockerfile_and_build.py

```diff
@@ -102,15 +102,14 @@ def create_build():
 RUN mkdir -p /workspace/build/fastertransformer_backend/build
 WORKDIR /workspace/build/fastertransformer_backend/build
 RUN cmake \\
-      -D CMAKE_EXPORT_COMPILE_COMMANDS=1 \\
-      -D CMAKE_BUILD_TYPE=Release \\
-      -D CMAKE_INSTALL_PREFIX=/opt/tritonserver \\
-      -D TRITON_COMMON_REPO_TAG="r${NVIDIA_TRITON_SERVER_VERSION}" \\
-      -D TRITON_CORE_REPO_TAG="r${NVIDIA_TRITON_SERVER_VERSION}" \\
-      -D TRITON_BACKEND_REPO_TAG="r${NVIDIA_TRITON_SERVER_VERSION}" \\
-      ..
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \\
+      -DCMAKE_BUILD_TYPE=Release \\
+      -DCMAKE_INSTALL_PREFIX=/opt/tritonserver \\
+      -DTRITON_COMMON_REPO_TAG="r{}" \\
+      -DTRITON_CORE_REPO_TAG="r{}" \\
+      -DTRITON_BACKEND_REPO_TAG="r{}" ..
 RUN make -j"$(grep -c ^processor /proc/cpuinfo)" install
-    '''
+    '''.format(FLAGS.triton_version, FLAGS.triton_version, FLAGS.triton_version)
     return df
```

Comment:
I think we should add additional logic here to change `triton_version` to default to the base container's triton version or something. There might be consequences in version mismatches? Like --base-image is 20.12 and --triton-version 22.12